docs/docs/_available-tags.md

```diff
@@ -0,0 +1,21 @@
+# Available tags
+
+The following is a list of available tags that can be used when contributing a sample. Click on the links to view the samples associated with the tags.
+
+Tag | Description
+-- | --
+[AI](./tags#AI) | Content dealing with AI related subjects. 
+[Cleanup](./tags#Cleanup) | Content about removing things and cleaning up.
+[Files](./tags#Files) | Content about dealing with files.
+[Guests](./tags#Guests) | Content about working with external users.
+[Lists](./tags#Lists) | Content involving lists or libraries.
```

Comment:
If we have tags for files, lists, sites, tasks, and users, shouldn't we have them for other entities as well, like flows, Teams teams, etc?